misc/dbt-materialize/mzcompose.yml

```diff
@@ -19,18 +20,49 @@ services:
       - MZ_DEV=1
     ports:
       - *materialized
+    depends_on: [test-certs]
```

Comment:
nit: I think this one is not required, because we don't use any certs with this service.
```suggestion
```

Reply:
ah - thank you!